src/components/pages/Discover.tsx

```diff
@@ -82,6 +82,36 @@ function convertCardArr(cardArr: DiscoverCardRaw[]): DiscoverCard[] {
 
 // TODO below JSON should be available from website.
 const DISCOVER_INFO = convertCardArr([
+  {
+    header: "Check out Raha Services",
```

Comment:
Why not marketplace?

Reply:
Yea maybe Marketplace is more clear ... trying to "nudge" people to offering Services but I agree wording is strange